docs/README.md

```diff
@@ -42,11 +42,13 @@ pip install recommonmark
 
 ## Building the documentation
 
-Make sure that there is a symlink from the `example` file (in /examples) inside the source folder. Run the following
-command to generate it:
+Make sure that there is a symlink from the `example` file (in /examples) and from the `CONTRIBUTING` guide (in the 
+root folder) inside the docs source folder. You shouldn't need to, but in case they are missing, run the following 
+commands to generate them:
 
 ```bash
 ln -s ../../examples/README.md examples.md
+ln -s ../../CONTRIBUTING.md contributing.md
```

Comment:
I wonder if these are necessary since the links are committed to the library... Just pulling the repo should automatically have the link files, shouldn't it?

Reply:
Yes, unless a person tries on windows (symlinks would need to be recreated) but since there are make commands after that don't work on Windows, I don't think we care too much.

Reply:
Right, and I don't think `ln` is a correct command to make links under Windows?

Reply:
Correct, that would `mklink` and I think the order is reversed. I deleted this part for now.